rules/external_binary/BUILD.bazel

```diff
@@ -1 +1,24 @@
 package(default_visibility = ["//visibility:public"])
+
+load(":def.bzl", "binary_location")
+
+[binary_location(
+    name = "{}_binary_location".format(name).replace("-", "_"),
+    binary = "@{}//:binary".format(name),
+) for name in [
+    "helm",
+    "jq",
+    "k3s",
+    "kind",
+    "kubectl",
+    "shellcheck",
+    "yaml2json",
+    "containerd",
+    "containerd-shim",
+    "ctr",
+    "docker",
+    "dockerd",
+    "docker-init",
+    "docker-proxy",
+    "runc",
```

Comment:
Sort?

Reply:
I actually wanted to keep the docker binaries groups. I'll add a comment.